index.js

```diff
@@ -73,20 +73,22 @@ const skeletorCli = () => {
 		return subTaskArgs.only.length > 0 && subTaskArgs.except.length > 0
 	};
 
-	const runTasks = (filteredTasks) => {
-		filteredTasks.forEach(task => {
-			skelCore.runTask(task.name, {
-				subTasksToInclude: filterSubTasks(task)
-			});
+	const runTask = task => {
+		skelCore.runTask(task.name, {
+			subTasksToInclude: filterSubTasks(task)
 		});
 	};
 
 	const filterTasks = () => {
+		let filteredTasks;
```

Comment:
[suggestion] I'm only mentioning this because you made a similar change above in the `runTask` method: this `filterTasks` method only returns one task (or null), so this variable name could be made singular.

Reply:
Good point -- fixed it.

Reply:
You did?

Reply:
Oops. Now I did.